### PR TITLE
Replace strlen with strnlen

### PR DESCRIPTION
**Pull request by amanCypherock**

### CODE REVIEW

**Comment by vbcypherock** [comment]
Task linked: [CHI-1972 Use strnlen instead of strlen](https://app.clickup.com/t/37308523/CHI-1972)

File: src/sdk_config.h [hunk 1]
```diff
@@ -86,7 +86,7 @@ extern atecc_interface_type atecc_mode;
 #define BSP_BUZZER_PWM_FREQ				4000
 #define BSP_BUZZER_TIMER_PRESCALAR		0
 #define BSP_BUZZER_TIMER_PERIOD			19999
-#define BSP_BUZZER_TIMER_COUNTER		3799
+#define BSP_BUZZER_TIMER_COUNTER		9999
```
Comment (ujjwal-cyph) [inline src/sdk_config.h:89]: unrelated change

**Review by ujjwal-cyph — CHANGES_REQUESTED** [review]

File: src/sdk_config.h (hunk 1 above)
Comment (ujjwal-cyph) [reply src/sdk_config.h:89]:
```suggestion
#define BSP_BUZZER_TIMER_COUNTER		3799
```

File: common/libraries/util/utils.c [hunk 2]
```diff
@@ -181,7 +181,7 @@ void __multi_to_single_line(const char input[24][15], const uint8_t number_of_mn
     uint16_t offset = 0U;
     uint16_t i = 0U;
     for (; i < number_of_mnemonics; i++) {
-        word_len = strlen(input[i]);
+        word_len = strnlen(input[i], 15);
```
Comment (ujjwal-cyph) [inline common/libraries/util/utils.c:184]:
```suggestion
        word_len = strnlen(input[i], 15);    // TODO: Define handling of overflow cases
```

**Review by ujjwal-cyph — APPROVED** [review]

**Review by ujjwal-cyph — APPROVED** [review]

**Pull request merged**